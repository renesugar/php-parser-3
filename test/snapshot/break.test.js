const parser = require('../main');

describe('break', () => {
  it('simple', () => {
    expect(parser.parseEval('break;')).toMatchSnapshot();
  });
  it('argument 0', () => {
    expect(parser.parseEval('break 0;')).toMatchSnapshot();
  });
  it('argument 1', () => {
    expect(parser.parseEval('break 1;')).toMatchSnapshot();
  });
  it('argument 2', () => {
    expect(parser.parseEval('break 2;')).toMatchSnapshot();
  });
});
