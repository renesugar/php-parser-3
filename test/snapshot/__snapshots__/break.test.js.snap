// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`break argument 0 1`] = `
Program {
  "children": Array [
    Break {
      "kind": "break",
      "level": Number {
        "kind": "number",
        "value": "0",
      },
    },
  ],
  "errors": Array [],
  "kind": "program",
}
`;

exports[`break argument 1 1`] = `
Program {
  "children": Array [
    Break {
      "kind": "break",
      "level": Number {
        "kind": "number",
        "value": "1",
      },
    },
  ],
  "errors": Array [],
  "kind": "program",
}
`;

exports[`break argument 2 1`] = `
Program {
  "children": Array [
    Break {
      "kind": "break",
      "level": Number {
        "kind": "number",
        "value": "2",
      },
    },
  ],
  "errors": Array [],
  "kind": "program",
}
`;

exports[`break simple 1`] = `
Program {
  "children": Array [
    Break {
      "kind": "break",
      "level": null,
    },
  ],
  "errors": Array [],
  "kind": "program",
}
`;
